.github/workflows/ci.yml

```diff
@@ -13,3 +13,32 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
       - uses: pre-commit/action@v2.0.3
+
+  test:
+    name: Test cases
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # Install Python.
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r app/requirements.txt
+          pip install pytest pytest-cov
+
+      - name: Run Test Cases
+        env:
+          MONGO_AUTH_CREDENTIALS: "mongodb+srv://quiz:p$#p4h7y_Z44R-n@cluster0.uocfg.mongodb.net/quiz?retryWrites=true&w=majority"
```

Comment:
why do we need this if we are running mongo mock?

Reply:
We will need this or else the post and get request will not be accepted, the mock needs to know which DB to mock.